.github/workflows/azure-nightly-build.yml

```diff
@@ -26,8 +29,14 @@ jobs:
 
   build-podvm-image:
     uses: ./.github/workflows/azure-podvm-image-build.yml
+
     needs:
     - generate-podvm-image-version
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+      attestations: write
```

Comment:
Do we need to specify the permissions here when they are specified in the "linked" `uses: ./.github/workflows/azure-podvm-image-build.yml` workflow? I mean previously they were not set here which would imply `permission={}` and it worked well.